.scripts/samples.js

```diff
@@ -177,6 +177,13 @@ const samples = [
     guide: '/api/extension-guides/custom-data-extension',
     apis: [],
     contributions: []
+  },
+  {
+    description: 'CodeLens Sample',
+    path: 'codelens-sample',
+    guide: null,
+    apis: [`languages.registerCodeLensProvider`, `CodelensProvider`, `CodeLens`],
```

Comment:
Should be `CodeLensProvider`

Reply:
done

Reply:
Sorry. If it wasn't clear, this was referring to line 185. Specifically, the change from `CodelensProvider` to `CodeLensProvider` -- `l` to `L`.

Reply:
Oh, I see. Should I revert back the change on line 182?